### PR TITLE
update footer responsive and select option contain icon by unicode. F…

### PR DESCRIPTION
**Pull request by codenuk**
…inally i import step progress bar to my page

### CODE REVIEW

File: src/components/upload-file/index.module.scss [hunk 1]
```diff
@@ -1,82 +1,93 @@
...
+    }
+
+    textarea {
+      border: 0.5px solid #009473;
+      resize: none;
     }
+  }
 }
+
+a {
```
Comment (Knight-H) [inline src/components/upload-file/index.module.scss:73]: wtf is this shit. use css modules

File: src/components/upload-file/index.module.scss [hunk 2]
```diff
@@ -1,82 +1,93 @@
...
-        border-radius: 2px;
-        padding: 5px 20px;
-        margin-bottom: 4vh;
-        svg {
-          width: 18px;
-          height: 18px;
-          margin-right: 10px;
+
+@media only screen and (max-width: 650px) {
+  body {
```
Comment (Knight-H) [inline src/components/upload-file/index.module.scss:16]: dangerous dude, but ok if you checked everything

File: src/components/upload-file/index.js [hunk 3]
```diff
@@ -1,43 +1,81 @@
...
+                                <p className={stylesDropDawn.selectBoxInputText}><i className="fas fa-check"></i>ต้องการอนุมัติ</p>
+                            </div>
+                            <div className={stylesDropDawn.selectBoxValue}>
+                                <input className={stylesDropDawn.selectBoxInput} type="radio" id="4" value="5" name="Ben" checked={dropDawn == 5 ? true : false}
+                                    onChange={(e) => handleChangeDropDawn(e.target)} />
+                                <p className={stylesDropDawn.selectBoxInputText}>กรุณาเลือก...</p>
+                                <IconArrow />
+                            </div>
+                        </div>
+                        <ul className={stylesDropDawn.selectBoxList}>
```
Comment (Knight-H) [inline src/components/upload-file/index.js:53]: dropdawn ?!?

File: src/components/upload-file/index.js [hunk 4]
```diff
@@ -1,43 +1,81 @@
...
+
+                <div className={styles.rightContent}>
+                    <label className={styles.label}>อนุมัติรูปแบบงาน</label>
 
-                <label>อัพโหลดไฟล์งาน</label>
+                    <div className={stylesDropDawn.selectBox}>
+                        <div className={stylesDropDawn.selectBoxCurrent} tabindex="1">
+                            <div className={stylesDropDawn.selectBoxValue}>
+                                <input className={stylesDropDawn.selectBoxInput} type="radio" id="1" value="2" name="Ben" checked={dropDawn == 2 ? true : false}
+                                    onChange={(e) => handleChangeDropDawn(e.target)} />
```
Comment (Knight-H) [inline src/components/upload-file/index.js:43]: why the hell is everything dropdawn

File: src/components/upload-file/index.js [hunk 5]
```diff
@@ -1,43 +1,81 @@
...
+                <img src={file} className={styles.square} />
+
+                <div className={styles.rightContent}>
+                    <label className={styles.label}>อนุมัติรูปแบบงาน</label>
 
-                <label>อัพโหลดไฟล์งาน</label>
+                    <div className={stylesDropDawn.selectBox}>
+                        <div className={stylesDropDawn.selectBoxCurrent} tabindex="1">
+                            <div className={stylesDropDawn.selectBoxValue}>
+                                <input className={stylesDropDawn.selectBoxInput} type="radio" id="1" value="2" name="Ben" checked={dropDawn == 2 ? true : false}
```
Comment (Knight-H) [inline src/components/upload-file/index.js:42]: what does dropdawn==2 mean, it cant be clearer?

**Review by Knight-H — CHANGES_REQUESTED** [review]
@SorawitSiri review please

**Comment by Knight-H** [comment]
+1 package of react-multi-carousel, please check if it is a good library to include

**Review by SorawitSiri — COMMENTED** [review]
We should reduce our code for CSS!!

File: src/components/upload-file/index.module.scss (hunk 1 above)
Comment (codenuk) [reply src/components/upload-file/index.module.scss:73]: Edit Done

File: src/components/upload-file/index.module.scss (hunk 2 above)
Comment (codenuk) [reply src/components/upload-file/index.module.scss:16]: Edit Done

File: src/components/upload-file/index.js (hunk 4 above)
Comment (codenuk) [reply src/components/upload-file/index.js:43]: if u want to merge stylesDropDawn in styles. i can do it

**Pull request closed without merging**